### PR TITLE
RMP-503 Added a prop to handle search key clearing on blur when addiing a button from outside via a slot

### PR DESCRIPTION
**Pull request by gayan-orangehrm**
RMP-501, 502 and 503 related vue app changes [here](https://websvn.orangehrm.com/orangehrm?op=comp&compare[]=/enterprise/branch/recruitment-vue-conversion/@170909&compare[]=/enterprise/branch/recruitment-vue-conversion/@170910)

### CODE REVIEW

File: components/src/core/components/Input/Autocomplete/AutocompleteInput.vue [hunk 1]
```diff
@@ -265,7 +269,7 @@ export default defineComponent({
       this.debouncer();
     },
     onBlur() {
-      if (!this.multiple && this.searchTerm) {
+      if (!this.multiple && this.searchTerm && !this.stopSearchKeyClearOnBlur) {
```
Comment (amilaw) [inline components/src/core/components/Input/Autocomplete/AutocompleteInput.vue:272]: @gayan-orangehrm Is this a requirement for the widget?  why would it need to remove the value on blur by default?

Comment (gayan-orangehrm) [reply components/src/core/components/Input/Autocomplete/AutocompleteInput.vue:272]: @amilaw it was passing `null` for `update:modelValue` from the very beginning of the component. checked in the OS code base as well.
Therefore I ignore it based on newly introduced prop `stopSearchKeyClearOnBlur`

Comment (gayan-orangehrm) [reply components/src/core/components/Input/Autocomplete/AutocompleteInput.vue]: Removed the code block which removed the value when on blur as per discussion with Chamara and some dev testings have done by myself after discussing with Chamara

Comment (Super-Chama) [reply components/src/core/components/Input/Autocomplete/AutocompleteInput.vue]: @gayan-orangehrm add the changelog.md file and add a record like below format.

`[Date] - [Commit hash] - [Affected Component] - [Remark]`

2022-03-29 -  7e7fca5 - AutocompleteInput.vue - Removed setting model value null onBlur. Possible affected components, QuickSearchInput.vue

**Pull request merged**